### ISSUE TITLE
San d'Oria Mission 3-3: Appointment to Jeuno (Bugged)

### ISSUE THREAD

**Issue by RoboRawb** [EdenServer/community#1455]
Currently unable to progress on this mission. 

Step 1: Acquired mission via gate guard as normal
Step 2: Spoke with Halver 
Step 3: Examined Door: Great Hall for CS and temporary key item: Letter to the Ambassador
Step 4: **Bugged** Speaking to Nelcabrit does **not** generate a CS

Character/IGN: Kaddis

**Comment by RoboRawb** [comment]
Update:

Per the advice of a GM, it was recommended to ignore this and give the mission a go regardless. 

As of this morning, we climbed the tower, acquired a Delkfutt's Tower Key and then proceeded to the basement where I examined the San d'Orian cement door at M-8. No CS was triggered. Trading the key to the door also did nothing in terms of converting it into a Key Item.

At this point, I am stuck on this mission with no way to proceed.
